### PR TITLE
chore(openapi): add redirects for OpenAPI docs

### PR DESCRIPTION
**Pull request by mboskamp**

### CODE REVIEW

**Review by tasso94 — APPROVED** [review]
Looks good to me 👍

File: config/live/.htaccess [hunk 1]
```diff
@@ -23,6 +23,457 @@ RewriteCond %{REQUEST_URI} ^/manual((?!7.18/)[^/]+)
 RewriteCond %{DOCUMENT_ROOT}/manual/%1 !-d
 RewriteRule ^manual/(.*) /manual/7.18/ [R=307,L]
 
+# OpenAPI docs
+RewriteRule ^manual/(latest|develop)/reference/rest/$ /rest/camunda-bpm-platform/7.19-SNAPSHOT/ [R=301,L,NE]
```
Comment (mboskamp) [inline config/live/.htaccess:27]:
```suggestion
RewriteRule ^manual/(latest|develop)/reference/rest/specification/$ /rest/camunda-bpm-platform/7.19-SNAPSHOT/ [R=301,L,NE]
```

File: config/stage/.htaccess [hunk 2]
```diff
@@ -18,6 +18,457 @@ RewriteCond %{REQUEST_URI} ^/manual((?!latest/)[^/]+)
 RewriteCond %{DOCUMENT_ROOT}/manual/%1 !-d
 RewriteRule ^manual/(.*) /manual/latest/ [R=307,L]
 
+# OpenAPI docs
+RewriteRule ^manual/(latest|develop)/reference/rest/$ /rest/camunda-bpm-platform/7.19-SNAPSHOT/ [R=301,L,NE]
```
Comment (mboskamp) [inline config/stage/.htaccess:22]:
```suggestion
RewriteRule ^manual/(latest|develop)/reference/rest/specification/$ /rest/camunda-bpm-platform/7.19-SNAPSHOT/ [R=301,L,NE]
```

**Review by tasso94 — CHANGES_REQUESTED** [review]
Great job! Looks good to me in general. Just two hints added below ⬇️

File: config/stage/.htaccess [hunk 3]
```diff
@@ -18,6 +18,457 @@ RewriteCond %{REQUEST_URI} ^/manual((?!latest/)[^/]+)
 RewriteCond %{DOCUMENT_ROOT}/manual/%1 !-d
 RewriteRule ^manual/(.*) /manual/latest/ [R=307,L]
 
+# OpenAPI docs
+RewriteRule ^manual/(latest|develop)/reference/rest/specification/$ /rest/camunda-bpm-platform/7.19-SNAPSHOT/ [R=301,L,NE]
```
Comment (mboskamp) [inline config/stage/.htaccess:22]:
```suggestion
RewriteRule ^manual/(latest|develop)/reference/rest/specification/(.*)$ /rest/camunda-bpm-platform/7.19-SNAPSHOT/$2 [R=301,L,NE]
RewriteRule ^manual/([^/]+)/reference/rest/specification/(.*)$ /rest/camunda-bpm-platform/$1/$2 [R=301,L,NE]
```

**Review by tasso94 — APPROVED** [review]

**Pull request merged**